app/src/main/java/ca/marshallasch/veil/FragmentCreatePost.java

```diff
@@ -0,0 +1,133 @@
+package ca.marshallasch.veil;
+
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.button.MaterialButton;
+import android.support.design.widget.Snackbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+
+import com.google.protobuf.Timestamp;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.UUID;
+
+import ca.marshallasch.veil.database.Database;
+import ca.marshallasch.veil.database.KnownHashesContract;
+import ca.marshallasch.veil.proto.DhtProto;
+import ca.marshallasch.veil.utilities.Util;
+
+/**
```

Comment:
Add in a class description. Other than that everything else looks good to go!

Reply:
Fixed in commit 5757038